libs/pika/async_mpi/CMakeLists.txt

```diff
@@ -41,3 +50,9 @@ pika_add_module(
     pika_config
   CMAKE_SUBDIRS examples tests
 )
+
+option(PIKA_WITH_MPI_GENERATE_SCRIPTS "Generate benchmarking scripts for mpi tests" OFF)
+mark_as_advanced(PIKA_WITH_MPI_GENERATE_SCRIPTS)
+if(PIKA_WITH_MPI_GENERATE_SCRIPTS)
+  add_subdirectory(scripts)
+endif()
```

Comment:
Can we leave this for another PR? The `scripts` subdirectory is not addeed here as far as I can tell.

Reply:
oops. that wasn't supposed to be committed. will remove and force push